### DISCUSSION TITLE
Тестирование сервисов ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#61] [Announcements]
# Тестирование сервисов ⚡️ Angular с примерами кода

Начнем с изучения тестирования сервисов Angular, поскольку именно сервисы проще всего покрываются тестами

https://angdev.ru/archive/angular9/services-unit-testing/

**Comment by FrooKKie** [comment]
TestBed.get() считается deprecated с 9 версии. В качестве альтернативы предлагается использовать TestBed.inject()
